### ISSUE TITLE
Keyboard focus boundaries have luminosity ratio of 1.2:1, which is less than required 3:1.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#23]
Attach (recommended) or Link to PDF file here:
https://mozilla.github.io/pdf.js/web/viewer.html.
[edad (2).pdf](https://github.com/gdh1995/pdf.js/files/14600923/edad.2.pdf)

Configuration:
- Operating system and its version: iPhone 13 Version-17.4
- PDF.js version: 1.4   

Steps to reproduce the problem:

1. Open this PDF viewer https://mozilla.github.io/pdf.js/web/viewer.html.
2. Navigate through the controls.
3. Observe the luminosity ratio of keyboard focus on pdf controls.

What is the expected behavior? (add screenshot)
Keyboard focus boundaries should have luminosity ratio of at least 3:1.

What went wrong? (add screenshot)
Keyboard focus boundaries have luminosity ratio of 1.2:1, which is less than required 3:1.
![Focus boundaries have low lumosity ratio (1)](https://github.com/gdh1995/pdf.js/assets/93735775/e4e09df9-fb6b-435e-89b3-d8194510dfd6)

The issue is repro'ing throughout the application for PDF controls.

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
